explain git restore (fixes #10) (#11)
Akuli/git-guide: committing.md

no changes added to commit (use "git add" and/or "git commit -a")
```

Adding, also known as staging, means choosing what will be included in the next commit.
This way it's possible to edit several files and commit only some of the changes.
Adding a file doesn't modify it;

> it just makes Git remember the changes you did and include them in the next commit.

Author: delete unnecessary repetition:

```suggestion
it just makes Git remember the changes you did.
```

+More text here. Lorem ipsum blah blah blah.
```

The first line of the README is `# reponame`, and it was not changed.
There was one line after that, `The description ...`, and it was deleted.
The rest was added when editing the file in the editor.

Diffing is optional, and it does not affect the output of `git status`,
but it may help you discover problems.
If you notice that something isn't quite right, you can still edit the files,
and then run `git add` again when you are done.
Alternatively, if you don't want to commit any changes to `README.md`,

> you can use the `git restore` command in `git status` output to undo the `git add`:

Author: ```suggestion
you can use `git restore` as shown in `git status` output to undo the `git add`:
```

```sh
$ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        modified:   README.md

$ git restore --staged README.md

$ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   README.md

no changes added to commit (use "git add" and/or "git commit -a")

$ git add README.md

$ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        modified:   README.md

```

Without `--staged`, `git restore` undoes changes that you saved in the editor but didn't `git add` yet.
This is useful when you realize that you wrote something stupid and you want to start over.
The same goes for `git diff`: without `--staged`,

> it shows changes that aren't added yet instead of what will be included in the commit.

Author: ```suggestion
instead of showing what will be included in the commit, it shows changes that aren't added yet.
```
